test/bdd/features/verifiable_credential.feature

```diff
@@ -38,6 +38,15 @@ Feature: Issue Verifiable Credential
 
 @interop_jwt_verifiable
   Scenario: Load and verify Interop JWT credentials
-    When loading interop credential number "1" and verify it
-    And  loading interop credential number "2" and verify it
-    And  loading interop credential number "3" and verify it
+    When loading interop credential number "1" signed using "Ed25519" and verify it
```

Comment:
what are the meaning of these numbers? can we have credential file name instead?

Reply:
sure, added